src/main/scala/io/iohk/ethereum/db/cache/LruCache.scala

```diff
@@ -1,18 +1,20 @@
 package io.iohk.ethereum.db.cache
 
 import java.util.concurrent.TimeUnit
-import java.util.concurrent.atomic.AtomicLong
+
 import io.iohk.ethereum.utils.Config.NodeCacheConfig
 import com.google.common.cache
 import com.google.common.cache.{CacheBuilder, RemovalNotification}
+
+import scala.jdk.CollectionConverters._
 import scala.concurrent.duration.FiniteDuration
 
 class LruCache[K <: AnyRef, V <: AnyRef](
     config: NodeCacheConfig,
     notificationHandler: Option[RemovalNotification[K, V] => Unit] = None
 ) extends Cache[K, V] {
 
-  private val lastClear = new AtomicLong(System.nanoTime())
+  @volatile private[this] var lastClear = System.nanoTime()
```

Comment:
Changing from:
`val lastClear = new AtomicLong`
to:
`@volatile private[this] var lastClear`
does not seems like good direction. Could you elaborate reason for this change?

Reply:
`Volatile` guarantees that each individual read and write ARE atomic per its definition.